src/main/scan/foreach.c

```diff
@@ -204,17 +204,8 @@ PyObject *AerospikeScan_Foreach(AerospikeScan *self, PyObject *args,
     }
 
     if (py_nodename) {
-        if (PyString_Check(py_nodename)) {
-            nodename = PyString_AsString(py_nodename);
-        }
-        else if (PyUnicode_Check(py_nodename)) {
-            py_ustr = PyUnicode_AsUTF8String(py_nodename);
-            if (!py_ustr) {
-                as_error_update(&data.error, AEROSPIKE_ERR_PARAM,
-                                "Invalid unicode nodename");
-                goto CLEANUP;
-            }
-            nodename = PyBytes_AsString(py_ustr);
```

Comment:
I think `py_ustr` is unused now. It and its cleanup should be deleted. This might have happened in other places where the PyUnicode_Check case switched to use PyUnicode_AsUTF8, I haven't noticed till now.

Reply:
Done